proxy/ProtocolProbeSessionAccept.cc

```diff
@@ -169,14 +169,20 @@ struct ProtocolProbeTrampoline : public Continuation, public ProtocolProbeSessio
 int
 ProtocolProbeSessionAccept::mainEvent(int event, void *data)
 {
+  static int no_activity_timeout = 0;
   if (event == NET_EVENT_ACCEPT) {
     ink_assert(data);
 
     VIO *vio;
     NetVConnection *netvc          = static_cast<NetVConnection *>(data);
     ProtocolProbeTrampoline *probe = new ProtocolProbeTrampoline(this, netvc->mutex, nullptr, nullptr);
 
-    // XXX we need to apply accept inactivity timeout here ...
+    // The connection has completed, set the accept inactivity timeout here to watch over the difference between the
+    // connection set up and the first transaction..
+    if (no_activity_timeout == 0) {
```

Comment:
Our doc says this config is "Reloadable", but this code doesn't look reloadable for me. 
We need to change either doc or code.
https://docs.trafficserver.apache.org/en/9.0.x/admin-guide/files/records.config.en.html#proxy-config-http-accept-no-activity-timeout